node/node.go

```diff
@@ -408,11 +407,12 @@ var emptyGetOptions = metav1.GetOptions{}
 
 // patchNodeStatus patches node status.
 // Copied from github.com/kubernetes/kubernetes/pkg/util/node
```

Comment:
Can you borrow the latest code from kubernetes upstream? https://github.com/kubernetes/kubernetes/blob/master/pkg/util/node/node.go#L175-L206

Reply:
Or: https://github.com/kubernetes/kubernetes/blob/master/pkg/util/node/node.go#L256-L268